<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cyrine</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>791</width>
        <height>531</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Fournisseurs</string>
       </attribute>
       <widget class="QTableView" name="tablefournisseur">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>761</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>781</width>
          <height>191</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Ajouter</string>
         </attribute>
         <widget class="QLineEdit" name="txt_cin">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>47</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom societe:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_nom">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>40</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Numero de tel: </string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_num">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>70</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>10</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Adresse E-maill:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_email">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>10</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>40</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Date d'adhesion:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>70</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Spécialité: </string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>680</x>
            <y>130</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_adresse">
          <property name="geometry">
           <rect>
            <x>620</x>
            <y>50</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>530</x>
            <y>50</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Adresse local:</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEdit">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>40</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_spec">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>70</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Modifier</string>
         </attribute>
         <widget class="QTableView" name="tableView_3">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>240</y>
            <width>781</width>
            <height>192</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>20</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Adresse E-maill:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_nom_soc">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>50</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Numero de tel: </string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>80</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Spécialité: </string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>60</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Adresse local:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Nom societe:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_tel_f">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>80</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>50</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Date d'adhesion:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_email_f">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>20</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_adresse_f">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>60</y>
            <width>113</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QDateEdit" name="dateEdit_2">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>50</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>47</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_id">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>20</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="txt_spc">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>80</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Supprimer</string>
         </attribute>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>50</y>
            <width>47</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>330</x>
            <y>40</y>
            <width>75</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="suppr_id">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>50</y>
            <width>113</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>460</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_suppr_f">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_pdf">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rech_bouton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_id">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>200</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_nom_soc">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>200</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom société</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_spec">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>200</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Spécialité</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_trier_fournisseur">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>230</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Trier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Commandes vente</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>771</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
